### PR TITLE
add dockerfile and docker run instructions

### PR DESCRIPTION
**Pull request by vordimous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ Optionally, you can pass a flag with the path to settings file:
 python3 mqtt-simulator/main.py -f <path/settings.json>
 ```
 
+Additionally, you can run via [Docker](https://docs.docker.com/get-docker/) with the included `Dockerfile`.
```
Comment (DamascenoRafael) [inline README.md:55]:
```suggestion
#### Running using Docker

Additionally, you can run via [Docker](https://docs.docker.com/get-docker/) with the included `Dockerfile`.
```

**Comment by DamascenoRafael** [comment]
Awesome @vordimous!! Thanks for your contribution 🎉
I've added a small suggestion to the README.
Other than that it looks good to me!

**Comment by vordimous** [comment]
@DamascenoRafael Thank you for looking at this! I almost made that extra header but figured you could if you wanted it, so that is perfect.

**Comment by vordimous** [comment]
Just a heads up, I plan to add an option in the settings.json file to include a separate data.json file to pull in a list of pre-defined messages instead of just simulating random ones. This is for a demo project I am working on, so I can add the functionality with a PR or leave it on my fork. Is that something you would want added to this project?

**Comment by DamascenoRafael** [comment]
@vordimous perfect!! Thanks again for the contribution 🚀 

As for the new additions, I'll be more than happy to review the changes in a future PR!
Using data from a separate file sounds nice, just add documentation on how to use it too.

**Pull request merged**